lib/components/map/connected-stop-viewer-overlay.js

```diff
@@ -62,7 +73,7 @@ class StopViewerOverlay extends MapLayer {
                 {...overlayConfig}
                 configCompanies={configCompanies}
                 stations={bikeRental && bikeRental.stations}
-                visible
+                visible={zoom >= MIN_ZOOM - 1}
```

Comment:
for my own curiosity: How does this component rerender when the zoom changes? I remember you saying that we couldn't get the zoom updates out of Leaflet, but that must be happening here. Is that because this is a layer that is installed in Leaflet?

Reply:
Good catch figuring out how this works! that's exactly it. We can only do `leaflet.map.getZoom()` here because we know that the render method will be called whenever Leaflet's zoom level changes. However, we can't pass this down to the stopSymbols so easily because we need to deal with the `ZoomBasedMarkers` sitting in-between. Not an impossible task, but definitely out of scope for this PR.